conans/client/build/cmake.py

```diff
@@ -232,14 +237,21 @@ def test(self, args=None, build_dir=None, target=None):
     @property
     def verbose(self):
         try:
-            verbose = self.definitions[verbose_definition_name]
-            return get_bool_from_text(str(verbose))
+            return get_bool_from_text(str(self._verbose))
```

Comment:
I would like to return just `self._verbose` and let the setter validate the input... any reason not to do it?